common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -339,6 +348,12 @@ def test_member_of_a_team(self):
             self.my_teams_page.visit()
         self.verify_teams(self.my_teams_page, teams)
 
+    def test_multiple_team_members(self):
+        teams = self.create_teams(self.topic, 1)
```

Comment:
Please add a BDD comment describing the test case (following the pattern of other test cases in the file).